homeassistant/components/plugwise/climate.py

```diff
@@ -143,8 +147,14 @@ def device_state_attributes(self):
         attributes = {}
         if self._outdoor_temperature:
             attributes["outdoor_temperature"] = self._outdoor_temperature
-        attributes["available_schemas"] = self._schema_names
-        attributes["selected_schema"] = self._selected_schema
+        if self._schema_names:
+            attributes["available_schemas"] = self._schema_names
+        else:
+            attributes["available_schemas"] = "None"
```

Comment:
If the item value is unknown or missing we don't add the item at all to state attributes.

Reply:
The STATES shows this:
![image](https://user-images.githubusercontent.com/11290930/73124504-675fc880-3f9c-11ea-976d-262a9b31df59.png)
The code is written like that to show that no schedule is selected, shown by the value 'None'.
Without the added code, it would show 'null', which I find unacceptable.

Reply:
From python perspective it would make sense, but for user experience it provides a clear 'you didn't create any schedules' indication.

Reply:
We shouldn't add the key either in the state attributes if the value is not set. So the item will not show at all.